Gen8/bdsp.py

```diff
@@ -10,6 +10,19 @@ def encounters():
     MAP_INFO = f"{SCRIPT_FOLDER}/bdsp/MapInfo.json"
     AREA_NAME = f"{SCRIPT_FOLDER}/bdsp/english_dp_fld_areaname.json"
     LOCATION_MODIFIERS = f"{SCRIPT_FOLDER}/location_modifier.json"
+    UG_ENCOUNT_02 = f"{SCRIPT_FOLDER}/bdsp/UgEncount_02.json"
+    UG_ENCOUNT_03 = f"{SCRIPT_FOLDER}/bdsp/UgEncount_03.json"
+    UG_ENCOUNT_04 = f"{SCRIPT_FOLDER}/bdsp/UgEncount_04.json"
+    UG_ENCOUNT_05 = f"{SCRIPT_FOLDER}/bdsp/UgEncount_05.json"
+    UG_ENCOUNT_06 = f"{SCRIPT_FOLDER}/bdsp/UgEncount_06.json"
+    UG_ENCOUNT_07 = f"{SCRIPT_FOLDER}/bdsp/UgEncount_07.json"
+    UG_ENCOUNT_08 = f"{SCRIPT_FOLDER}/bdsp/UgEncount_08.json"
+    UG_ENCOUNT_09 = f"{SCRIPT_FOLDER}/bdsp/UgEncount_09.json"
+    UG_ENCOUNT_10 = f"{SCRIPT_FOLDER}/bdsp/UgEncount_10.json"
+    UG_ENCOUNT_11 = f"{SCRIPT_FOLDER}/bdsp/UgEncount_11.json"
+    UG_ENCOUNT_12 = f"{SCRIPT_FOLDER}/bdsp/UgEncount_12.json"
+    UG_ENCOUNT_20 = f"{SCRIPT_FOLDER}/bdsp/UgEncount_20.json"
+    UG_SPECIAL_POKEMON = f"{SCRIPT_FOLDER}/bdsp/UgSpecialPokemon.json"
```

Comment:
General comment here reviewing what was done in gen 4, this should all be moved to a underground function